### PR TITLE
UNI-1899 Credit cub ens/avatar overrides

### PR DESCRIPTION
**Pull request by 0xbarbs**

### CODE REVIEW

File: src/constants.js
Comment (kingjacob) [inline src/constants.js:1]: We deleted goerli and opgoerli, does this add it back?

**Review by kingjacob — COMMENTED** [review]
Wheres the file that holds the custom addresses and pfps?

File: src/constants.js
Comment (kingjacob) [reply src/constants.js:1]: @0xbarbs

Comment (0xbarbs) [reply src/constants.js:1]: no it just looks like there are some goerli constants that weren't removed

**Review by kingjacob — APPROVED** [review]

**Pull request merged**